File: Backend/Models/WorkOrder.cs
namespace Backend.Models;

public class WorkOrder
{
    public string? WorkOrderNo { get; set; }
    public string? MachineNo { get; set; }
    public string? OperatorName { get; set; }
    public int OrderQty { get; set; }
    public int CompletedQty { get; set; }
} 

File: Frontend/src/types/WorkOrder.ts
export interface WorkOrder {
  workOrderNo: string;
  machineNo: string;
  operatorName: string;
  orderQty: number;
  completedQty: number;
}

export interface WorkOrderRequest {
  WorkOrderNo: string;
  MachineNo: string;
  OperatorName: string;
  OrderQty: number;
  CompletedQty: number;
}

export interface ApiResponse<T> {
  data: T | null;
  error: string | null;
  loading: boolean;
}